pkg/core/native/notary.go

```diff
@@ -39,17 +39,22 @@ type Notary struct {
 	// blockchain DAO persisting. If true, we can safely use cached values.
 	isValid                bool
 	maxNotValidBeforeDelta uint32
+	notaryServiceFeePerKey int64 // TODO: make it uint64?
```

Comment:
@roman-khimov, we need to decide about several TODOs, currently, I've followed the C# behaviour, but it can be improved.

Reply:
BTW, I don't think `int64` limits us in any way here.